frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-readonly-session/create-readonly-session-dialog.component.html

```diff
@@ -56,11 +55,29 @@ <h2>Open read-only Session</h2>
         administrator or project lead.
       </div>
     }
+
+    @if (
+      maxNumberOfModels &&
+      this.selectedModelOptions().length > maxNumberOfModels
+    ) {
+      <div class="my-1 border p-2 text-sm shadow">
+        <p class="text-error">
+          The selected tool model only supports up to
+          {{ maxNumberOfModels }} session at a time.
```

Comment:
```suggestion
          The selected tool only supports up to
          {{ maxNumberOfModels }} provisioned models at a time.
```

The initial sentence doesn't make much sense.

Reply:
Done